api/database/tables/sponsorships.go

```diff
@@ -0,0 +1,9 @@
+package tables
+
+type Sponsorships struct {
+	ID          int    `gorm:"column:id;primaryKey;autoIncrement"`
+	PocID       int    `gorm:"column:poc_id;not null"`
+	Email       string `gorm:"column:email;type:varchar(100)"`
+	Status      string `gorm:"column:status;not null;type:varchar(100)"`
+	Description string `gorm:"column:description;type:varchar(500)"`
+}
```

Comment:
have an option to have multiple emails

Also need another option to see who submitted the sponsorship

Reply:
Updated that to CurrentEmail, other emails will be stored in Updates field
> who submitted the sponsorship
thats PocID, it will be autofilled in the frontend by the logged in user

Reply:
where are you storing the company name, company PoCs, their contacts etc